reference/constraints/Email.rst

```diff
@@ -144,6 +144,7 @@ If true, then the :phpfunction:`checkdnsrr` PHP function will be used to
 check the validity of the MX *or* the A *or* the AAAA record of the host
 of the given email.
 
+.. include:: /reference/constraints/_ltrim-rtim-options.rst.inc
```

Comment:
I would prefer 2 files:
`/reference/constraints/_ltrim-option.rst.inc`
and
`/reference/constraints/_rtrim-option.rst.inc`